mrmustard/lab_dev/wires.py

```diff
@@ -111,7 +112,7 @@ def _from_data(cls, id_array=None, modes=None, mask=None):
         w = cls()
         w._id_array = id_array if id_array is not None else w._id_array
         w._modes = modes if modes is not None else w._modes
-        w._mask = mask if mask is not None else w._mask
+        w._mask = mask if mask is not None else np.ones_like(w._id_array)
```

Comment:
In case you missed it:
https://github.com/XanaduAI/MrMustard/pull/324#discussion_r1467737055

Reply:
weird, I thought I had changed this